app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -4,7 +4,7 @@ class CodeSchoolsController < ApiController
       before_action :authenticate_user!, only: [:create, :update, :destroy]
 
       def index
-        render json: CodeSchool.order(:name)
+        render json: CodeSchool.order(:name).includes(:locations)
```

Comment:
Before passing the query results to the JSON rendering in the view layer, include the locations associated with schools in the results.